core/build.gradle

```diff
@@ -21,6 +21,7 @@ managedDependencies {
         compile "io.netty:$it"
     }
     compile 'io.netty:netty-transport-native-epoll:linux-x86_64'
+    compile 'io.netty:netty-transport-native-kqueue:osx-x86_64'
```

Comment:
I guess it doesn't hurt having them although it will add a little bit of bloat. Isn't it better than letting user add a platform-specific transport JAR explicitly given its perf boost?